content/en/docs/reference/command-line-tools-reference/feature-gates/ContainerStopSignals.md

```diff
@@ -0,0 +1,13 @@
+---
+title: ContainerStopSignals
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.33"
+---
+Enables usage of the StopSignal lifecycle hook for containers for configuring custom [stop signals](https://docs.docker.com/reference/dockerfile/#stopsignal) using which the containers would be stopped.
```

Comment:
Don't link to Docker documentation; link to Kubernetes docs instead.

Reply:
Since we don't have a dedicated page for stop signals yet, I'll link this to the StopSignal section in the container lifecycle hook page above. Should we add a reference page for all the [supported signals](https://github.com/sreeram-venkitesh/kubernetes/blob/52c3010c8b7864604cda2d0f9d9f139b8014ddb1/pkg/apis/core/types.go#L2680-L2746)?

Reply:
See https://github.com/kubernetes/website/pull/49857#discussion_r1966902629 for the steps I recommend.  I don't think we need a reference page, but you can add one if you want.